site/src/theme/permission.ts

```diff
@@ -0,0 +1,5 @@
+export interface Permission {
+	background: string;
+	outline: string;
+	text: string;
+}
```

Comment:
Imho, this is way too specific of a thing to add to the theme. We've been missing a clear "neutral" `role` choice for a while, and I've been meaning to resolve that for a while but it just hasn't come up recently. That being said, you got me thinking about it again today and I have proposed a fix in #14356. If that PR gets merged, I'd much rather use `theme.roles.info` colors for these than complicate the global `Theme` for this one specific use case.

Reply:
I really just added the Permission theme to have a short discussion about it. I was originally going to use or modify role but semantically permissions are not roles and if we are using a feature concept such as roles in the theme it would make sense to add permissions as well. And this will also make it more likely for others in the future to have the idea to add theming for features (like I did) then adding theming for UI concepts. For example, this could happen with other elements that have a lot of variations such as avatars, buttons, cards, status indicators, etc.

The other way to do it is more like in mui.ts where its about UI elements and tags. If we following that line of thinking then we could use the name pills in the theme instead of roles. In this case we are thinking more about the UI component itself than the feature it is used for.

Reply:
Sorry about the confusion. The `theme.roles` property is entirely unrelated to the "Roles" feature in the product. 😄

Reply:
theme.roles.info.background and theme.roles.info.outline are different than the colors I want to use. Where would you suggest placing the colors I wanted to use from theme.roles.default?